lib/pbench/test/unit/server/query_apis/test_search.py

```diff
@@ -119,6 +83,9 @@ def test_query(
                                 "controller": "dhcp31-171.perf.lab.eng.bos.redhat.com",
```

Comment:
This code will be checked into a publicly-available repo (right?), and we're supposed to guard against leaking information about internal networks, hostnames, and usernames (right?), so this should be something like `dhcp1-1.example.com`.

And, the same comment applies to the myriad other places where you have example or test data.  🙁

Reply:
When you make this change, please make it in its own commit....  😛